CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - The ampersand checker now skips verbatim fields (`file`, `url`, ...). [#10419](https://github.com/JabRef/jabref/pull/10419)
 - If no existing document is selected for exporting "XMP annotated pdf" JabRef will now create a new PDF file with a sample text and the metadata. [#10102](https://github.com/JabRef/jabref/issues/10102)
 - We modified the DOI cleanup to infer the DOI from an ArXiV ID if it's present. [10426](https://github.com/JabRef/jabref/issues/10426)
+- The Medline importer uses the field `comment` for notes (instead of `review).
```

Comment:
```suggestion
- The ISI importer uses the field `comment` for notes (instead of `review).
```

Reply:
The code read as follows

> * Importer for the ISI Web of Science, INSPEC and Medline format.

Maybe, that comment is wrong somewhow? I am not that into the topic of Medline vs. ISI Web of Science

Reply:
It's a bit misleading: I would say for the ISI web of science/INSPEC format 
https://webofscience.help.clarivate.com/Content/export-records.htm